site/app/templates/HomePage.twig

```diff
@@ -56,6 +56,28 @@
                     </a>
                 </li>
             {% endif %}
+            <li>
+                <br>
+                <button title="Enable Dark Mode" class="btn btn-block dark_mode_toggle key_to_click" onclick="toggleTheme();" style="background-color: var(--dark-mode-black); color: var(--text-white)">
```

Comment:
can we move the style attributes here into the `dark_mode_toggle` class (and same with the other buttons and their style attributes.